ui/app/send-v2.js

```diff
@@ -445,6 +449,8 @@ SendTransactionScreen.prototype.onSubmit = function (event) {
     txParams.to = to
   }
 
+  clearSend()
```

Comment:
Why is this in `onSubmit`?  If we ever need to reuse state between submits (error handling maybe), would this be limiting?  I'm thinking of how "when the user leaves a screen" is often synonymous with "stuff that happens on lifecycle/unmount.

Reply:
Good point. One option would be to clear it on mount. Then we will always have the last sent transactions data in state up until the point when we need to clear it to create a new one.

Reply:
@danjm - yep, I like it.  Feel free to add it to next PR.